sharktank/tests/ops/ops_test.py

```diff
@@ -35,26 +34,6 @@ def testBroadcastDims(self):
         assert res[1] == 2
 
 
-class ElementwiseTest(unittest.TestCase):
```

Comment:
Why are we no longer testing?

Reply:
This tests the binary op + folding, which is no longer present.